src/lwc/gsApplicationStatus/__tests__/gsApplicationStatus.test.js

```diff
@@ -0,0 +1,94 @@
+
+let dayIn15Days = new Date();
+dayIn15Days = dayIn15Days.setDate(dayIn15Days.getDate() + 15);
+
+// Mock data for a trial org that expires in 15 days and has not applied yet
+const NOT_APPLIED_EXPIRATION_DATE_15_DAYS = {
+    isSandbox: false,
+    trialExpirationDate : dayIn15Days,
+    applicationDate: null
+}
+// Mock data for a trial org that expires in 15 days and has applied.
+const APPLIED_EXPIRATION_DATE_15_DAYS = {
+    isSandbox: false,
+    trialExpirationDate : dayIn15Days,
+    applicationDate: new Date()
+}
+
+describe('c-application-status', () => {
+    afterEach(() => {
+        while(document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    /**
+     * @description testing 15 days left and no application
+     * Asserts:
+     *     * button is apply for free licenses
+     *     * 15 days trial org remaining
+     *     * Img is the one that correponds with the this state
+     */
+    it('shows apply button and shows 15 days left when application is not applied', () => {
+        getApplicationStatus.mockResolvedValue(NOT_APPLIED_EXPIRATION_DATE_15_DAYS);
+        const element = createElement('c-gs-application-status', {is: GsApplicationStatus});
+        document.body.appendChild(element);
+        return flushPromises().
+        then(() => {
+            const button = element.shadowRoot.querySelector('.slds-button');
+            expect(button.innerHTML).toBe('c.gsApplyForFreeLicenses');
+            const daysLeft = element.shadowRoot.querySelector(".daysLeft");
+            expect(daysLeft.innerHTML).toContain('15');
+            const img = element.shadowRoot.querySelector('img').getAttribute('src');
+            expect(img).toBe('CumulusStaticResources/gsResources/Accept_Tasks_Apply_Card.png');
+
+        })
+        
+    });
+
+    /**
+     * @description testing 15 days left and application
+     * Asserts:
+     *     * button is apply for free licenses
+     *     * 15 days trial org remaining
+     *     * Img is the one that correponds with the this state
+     */
+    it('shows check status button and shows 15 days left when application is not applied', () => {
+        getApplicationStatus.mockResolvedValue(APPLIED_EXPIRATION_DATE_15_DAYS);
+        const element = createElement('c-gs-application-status', {is: GsApplicationStatus});
+        document.body.appendChild(element);
+        return flushPromises().
+        then(() => {
+            const button = element.shadowRoot.querySelector('.slds-button');
+            expect(button.innerHTML).toBe('c.gsCheckStatus');
+            const daysLeft = element.shadowRoot.querySelector(".daysLeft");
+            expect(daysLeft.innerHTML).toContain('15');
+            const img = element.shadowRoot.querySelector('img').getAttribute('src');
+            expect(img).toBe('CumulusStaticResources/gsResources/gift_illustration_2.svg');
+        })
+        
+    });
+});
```

Comment:
Could you import and run `sa11y`? https://developer.salesforce.com/blogs/2020/10/automated-accessibility-testing-with-sa11y.html

Reply:
I added sa11y and locally it works. But i broke the build on github.